crypto/evp/e_aes.c

```diff
@@ -2686,8 +2694,12 @@ static int aes_init_key(EVP_CIPHER_CTX *ctx, const unsigned char *key,
         ret = vpaes_set_encrypt_key(key, EVP_CIPHER_CTX_key_length(ctx) * 8,
                                     &dat->ks.ks);
         dat->block = (block128_f) vpaes_encrypt;
-        dat->stream.cbc = mode == EVP_CIPH_CBC_MODE ?
-            (cbc128_f) vpaes_cbc_encrypt : NULL;
+        dat->stream.cbc = NULL;
+        if (mode == EVP_CIPH_CBC_MODE)
+            dat->stream.cbc = (cbc128_f) vpaes_cbc_encrypt;
+        else
+        if (mode == EVP_CIPH_CTR_MODE)
```

Comment:
Huh, is this the usually OpenSSL style? I thought you all usually had `else` and `if` on the same line.

Reply:
Okay, I will look if the same style is nearby, I think that I would not break the local style
since style-cleanups tend to cause merge conflicts, but yes, sometimes I mix up the
style guides from openssl gcc linux OPC/UA as I often switch between different styles ;-)